src/main/java/com/google/sps/model/ActionableMessage.java

```diff
@@ -63,4 +67,38 @@ public boolean equals(Object obj) {
         .append(subject, actionableMessage.getSubject())
         .isEquals();
   }
+
+  /**
+   * Compares two actionable messages by their priorities. High priority > Medium priority > Low
+   * priority. If the assigned priorities are the same, the actionableMessage that was sent later is
+   * considered to be higher priority
+   *
+   * @param actionableMessage the ActionableMessage that this object is being compared against
+   * @return A positive integer if the current object has the higher priority, a negative integer if
+   *     the passed object has the higher priority, 0 if both objects have the same priority
+   */
+  @Override
+  public int compareTo(ActionableMessage actionableMessage) {
+    if (priority.getPriorityValue() == actionableMessage.priority.getPriorityValue()) {
+      return (int) (internalDate - actionableMessage.internalDate);
```

Comment:
Is it possible to get away with not casting the difference in `internalDates` to `int` and keeping them as `long`?

Reply:
Unfortunately no, since `compareTo` takes an integer as a return value. It's pretty cool how this works - if you return a positive integer, it means the current object is larger. If you return negative, the passed in object is larger. If you return 0, they are equal. 

The line you are referring to is just shorthand for "the email sent later is considered "larger" for sorting purposes"